services/app-web/src/pages/App/App.tsx

```diff
@@ -16,18 +16,28 @@ import TraceProvider from '../../contexts/TraceContext'
 import { AuthModeType } from '../../common-code/config'
 import { S3Provider } from '../../contexts/S3Context'
 import type { S3ClientT } from '../../s3'
+import { useScript } from '../../hooks'
+import { generateNRScriptContent } from '../../newRelic'
+import { getEnv } from '../../configHelpers/envHelpers'
+
 
 export type AppProps = {
     authMode: AuthModeType
     apolloClient: ApolloClient<NormalizedCacheObject>
     s3Client: S3ClientT
 }
 
+
 function App({
     authMode,
     apolloClient,
     s3Client,
 }: AppProps): React.ReactElement {
+    const environmentName = process.env.REACT_APP_STAGE_NAME || ''
+    const isHigherEnv = ['prod', 'val', 'dev'].includes(environmentName )
```

Comment:
`STAGE_NAME` in dev is confusingly `main`, so we'll want that in this array. Really not sure why it was called `main` 🤷🏻‍♂️